plugin/EasyGrep.vim

```diff
@@ -1973,6 +1973,11 @@ function! s:ParseCommandLine(argv)
     let opts["failedparse"] = ""
     let parseopts = 1
 
+    if exists("g:EasyGrepDisableCmdParam") && g:EasyGrepDisableCmdParam==1
```

Comment:
I don't think we need an option for this. EasyGrep supports adding -- to the command line which says that "everything after --" is taken as the pattern.